lib/engines/hcktest/tests.rb

```diff
@@ -223,7 +223,7 @@ def report_data
         'system_info' => {
           'guest' => @clients_system_info
         },
-        'sections' => RESULTS_REPORT_SECTIONS - @project.options.test.reject_report_sections
+        'sections' => RESULTS_REPORT_SECTIONS - @project.options.test.reject_report_sections.to_a
```

Comment:
Let's assign `[]` to @project.options.test.reject_report_sections just as we did for `AutoHCK::CLI::InstallOptions#drivers`.